main.go

```diff
@@ -172,7 +180,22 @@ summary {
 				os.Remove(filename)
 				w.Write([]byte(fmt.Sprintf("removed %s", filename)))
 			} else {
-				http.ServeFile(w, r, filename)
+				v, ok := r.URL.Query()["rename"]
+				if ok && v[0] == "true" {
```

Comment:
My first golang! Not sure if there is a better pattern than this nested if :shrug: